File: find_diff.py
def findDifference(self, nums1: List[int], nums2: List[int]) -> List[List[int]]:
    list1 = set([x for x in nums1 if x not in nums2])
    list2 = set([x for x in nums2 if x not in nums1])
    return [list1,list2]

File: findDiff.js
var findDifference = function(nums1, nums2) {
    let arr1 = nums1.filter(x=> !nums2.includes(x));
    let arr2 = nums2.filter(x=> !nums1.includes(x));
    arr1 = _.uniq(arr1,false);
    arr2 = _.uniq(arr2,false);
    return [arr1,arr2];
};